File: StarWars/APICORE/Contracts/Product.cs
using System.ComponentModel.DataAnnotations;

namespace APICORE.Contracts
{
    public class Product
    {
        [Required]
        public string Title { get; set; }

        [Required]
        public int Price { get; set; }

        [Required]
        public string Zipcode { get; set; }

        [Required]
        public string Seller { get; set; }

        [Required]
        public string ThumbnailHd { get; set; }

        [Required]
        public string Date { get; set; }
    }
}

File: desafiostone/src/main/java/br/com/stone/desafiostone/entity/Product.java
package br.com.stone.desafiostone.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "tb01_products")
@Getter
@Setter
public class Product {

  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private long id;
  private String title;
  private int price;
  private String zipcode;
  private String seller;
  private String thumbnailHd;
  private String Date;

}
